put cmake files in share/cmake instead of lib/cmake
Author: Rex Dieter <rdieter@fedoraproject.org>
diff -up websocketpp-0.7.0/CMakeLists.txt.cmake_noarch websocketpp-0.7.0/CMakeLists.txt
--- websocketpp-0.7.0/CMakeLists.txt.cmake_noarch	2016-02-22 07:30:10.000000000 -0600
+++ websocketpp-0.7.0/CMakeLists.txt	2016-09-15 13:05:59.012179625 -0500
@@ -19,7 +19,7 @@ set(INSTALL_INCLUDE_DIR include CACHE PA
 if (WIN32 AND NOT CYGWIN)
   set (DEF_INSTALL_CMAKE_DIR cmake)
 else ()
-  set (DEF_INSTALL_CMAKE_DIR lib/cmake/websocketpp)
+  set (DEF_INSTALL_CMAKE_DIR share/cmake/websocketpp)
 endif ()
 set (INSTALL_CMAKE_DIR ${DEF_INSTALL_CMAKE_DIR} CACHE PATH "Installation directory for CMake files")
 
